### ISSUE TITLE
URLs between cropLimit/2 and cropLimit are not cropped properly

### ISSUE THREAD

**Issue by Loirooriol** [devtools-html/devtools-core#875]
```js
Rep({
  object: "http://example.com/foobarbaz",
  cropLimit: 50,
}).props.children[1].props.children
```

Result: `"http://example.com/fooba…://example.com/foobarbaz"`
Expected: `"http://example.com/foobarbaz"`

Regressed by [bug 1419479](https://bugzilla.mozilla.org/show_bug.cgi?id=1419479), probably https://github.com/devtools-html/devtools-core/commit/e9238260b69412731d3f158444be5a10eb8f2658. Causes [bug 1427095](https://bugzilla.mozilla.org/show_bug.cgi?id=1427095).

Cc @nchevobbe

**Comment by jasonLaster** [comment]
this might be similar as the URLs encoding is also off here
https://github.com/devtools-html/debugger.html/issues/4902

**Comment by nchevobbe** [comment]
> this might be similar as the URLs encoding is also off here
devtools-html/debugger.html#4902

I don't get what is your issue jason ?

**Comment by nchevobbe** [comment]
Sorry about that @Loirooriol . The fix will be part of a devtools-reps-0.18.0 bundle
